packages/common/src/content/index.ts

```diff
@@ -287,6 +288,8 @@ export function datasetToContent(dataset: DatasetResource): IHubContent {
     recordCount,
     statistics,
     // additional attributes needed
+    extent,
+    itemExtent,
```

Comment:
Do you need `itemExtent`? It seems like you just end up referencing `item.extent` down below.